apps/assessment_registry/dataloaders.py

```diff
@@ -29,6 +31,63 @@ def batch_load_fn(self, keys):
         return Promise.resolve([_map.get(key) for key in keys])
 
 
+class AssessmentRegistryIssueChildLoader(DataLoaderWithContext):
+    def batch_load_fn(self, keys):
+        qs = SummaryIssue.objects.filter(
+            parent__in=keys
+        ).values(
+            'parent'
+        ).annotate(
+            child_count=Count(
+                'parent'
+            )
```

Comment:
Can you confirm if this is working?
We will need to count 'id' instead of 'parent', I think right now all value are either 0 or 1.
```suggestion
            child_count=Count(
                'id'
            )
```

Reply:
@thenav56 dai,  this is working fine.

Reply:
By working, can you check if the values are correct?